src/components/TaskView.js

```diff
@@ -93,6 +93,13 @@ function TaskView({ events, setEvents, user }) {
                   <TaskCard event={event} zoomFilter={zoomFilter} user={user} />
                 </div>
               ))}
+              <div
+                id="column"
+                className="column"
+                style={{ marginLeft: 250, marginRight: 250 }}
+              >
```

Comment:
Centering should be done via a container or level and preferably inside of a section.
Something like:
```            
<div className="section">
    <div className="level">
        <div className="level-item">
            <AddTask user={user} />
        </div>
    </div>
 </div>